monkestation/code/modules/bloodsuckers/monster_hunters/hunter_datum.dm

```diff
@@ -17,20 +20,39 @@
 	var/apocalypse = FALSE
 	///a list of our prey
 	var/list/datum/mind/prey = list()
+	/// A list of traits innately granted to monster hunters.
+	var/static/list/granted_traits = list(
+		TRAIT_FEARLESS,
+		TRAIT_HARDLY_WOUNDED,
+		TRAIT_NOCRITDAMAGE,
+		TRAIT_NOSOFTCRIT,
+		TRAIT_NO_ZOMBIFY,
+		TRAIT_TRUE_NIGHT_VISION
+	)
```

Comment:
this is a stealth change and a very powerful one at that.

Reply:
(they had `TRAIT_NOSOFTCRIT` and `TRAIT_NOCRITDAMAGE` before tho, to be fair)